plugin.js

```diff
@@ -1,17 +1,18 @@
 'use strict'
 
 const fp = require('fastify-plugin')
+const compare = require('secure-compare')
 
-const internals = {}
-internals.factory = function (options) {
+function factory (options) {
   const defaultOptions = {
-    keys: new Set(),
+    keys: [],
```

Comment:
Why is this no longer a `Set`? At the very least, the documentation needs to be updated to reflect this change.

Reply:
no, it's compatible with `Set` and `arrays`. A set and array have different interfaces.

Reply:
@jsumners I updated the docs